src/Forms/FieldList.php

```diff
@@ -42,13 +45,14 @@ public function __construct($items = [])
         parent::__construct($items);
 
         foreach ($items as $item) {
-            if ($item instanceof FormField) {
-                $item->setContainerFieldList($this);
+            if (!is_a($item, FormField::class)) {
+                throw new InvalidArgumentException('Item is not a FormField, is of class ' . get_class($item));
```

Comment:
Note this is kind of unnecessary because setContainerFieldList() only exists on FormField so would fail there, so it's just a little cleaner given the phpdoc signature